src/Altinn.Platform/Altinn.Platform.Events/Events/Repository/EventsPostgresRepository.cs

```diff
@@ -0,0 +1,55 @@
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Altinn.Platform.Events.Configuration;
+using Altinn.Platform.Events.Models;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Npgsql;
+
+namespace Altinn.Platform.Events.Repository
+{
+    /// <summary>
+    /// Handles events repository. 
+    /// </summary>
+    [ExcludeFromCodeCoverage]
+    public class EventsPostgresRepository : IEventsPostgresRepository
```

Comment:
Could we simplify the name to be just PostgresRepository ? Not likely that we will be storing anything other than events in postgres for now, but the word Events doesn't give that much value

Reply:
Oh yes! To much "Events" in this code :)